packages/elements-core/src/components/Docs/HttpOperation/Parameters.tsx

```diff
@@ -62,17 +62,23 @@ export const Parameter: React.FunctionComponent<IParameterProps> = ({ parameter,
   // TODO (JJ): schema.deprecated is used in platform - to be removed once it's updated https://github.com/stoplightio/platform-internal/issues/2267
   const deprecated = get(parameter, 'deprecated') || get(parameter, 'schema.deprecated', false);
 
+  const parameterExamples =
+    parameter.examples?.map(example => {
+      if (isNodeExample(example)) {
+        return example.value;
+      }
+
+      return example.externalValue;
+    }) || [];
+
+  const schemaExamples = parameter.schema?.examples;
+  const schemaExamplesArray = Array.isArray(schemaExamples) ? schemaExamples : [];
```

Comment:
Why not just `const schemaExamples = parameter.schema?.examples ?? []`, like above? Are we ever interested in these variables by themselves?

Reply:
Because TS complains, since typings for that JSONSchema7 type are not obeying the spec and are claiming that something else other than array can be there. 

Meanwhile the spec clearly states this must be an array of values. 🤷

Reply:
I get it 🤔